/media/data_dump_1/arushi_mca/GAN_try/AttnGAN/code/miscc/config.py:103: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  yaml_cfg = edict(yaml.load(f))
/home/arushi_mca/.conda/envs/MCAProject/lib/python2.7/site-packages/torchvision/transforms/transforms.py:220: UserWarning: The use of the transforms.Scale transform is deprecated, please use transforms.Resize instead.
  "please use transforms.Resize instead.")
THCudaCheck FAIL file=/pytorch/torch/csrc/cuda/Module.cpp line=59 error=101 : invalid device ordinal
Using config:
{'B_VALIDATION': False,
 'CONFIG_NAME': 'attn2',
 'CUDA': True,
 'DATASET_NAME': 'recipe',
 'DATA_DIR': '../data/recipe',
 'GAN': {'B_ATTENTION': True,
         'B_DCGAN': False,
         'CONDITION_DIM': 100,
         'DF_DIM': 96,
         'GF_DIM': 48,
         'R_NUM': 3,
         'Z_DIM': 100},
 'GPU_ID': 2,
 'RNN_TYPE': 'LSTM',
 'TEXT': {'CAPTIONS_PER_IMAGE': 1, 'EMBEDDING_DIM': 188, 'WORDS_NUM': 12052},
 'TRAIN': {'BATCH_SIZE': 100,
           'B_NET_D': False,
           'DISCRIMINATOR_LR': 0.0002,
           'ENCODER_LR': 0.0002,
           'FLAG': False,
           'GENERATOR_LR': 0.0002,
           'MAX_EPOCH': 101,
           'NET_E': '../DAMSMencoders/recipe/text_encoder50.pth',
           'NET_G': '../models/netG_epoch_101.pth',
           'RNN_GRAD_CLIP': 0.25,
           'SMOOTH': {'GAMMA1': 5.0,
                      'GAMMA2': 5.0,
                      'GAMMA3': 10.0,
                      'LAMBDA': 1.0},
           'SNAPSHOT_INTERVAL': 2000},
 'TREE': {'BASE_SIZE': 64, 'BRANCH_NUM': 3},
 'WORKERS': 1}
Load filenames from: ../data/recipe/train/filenames.pickle (10000)
Load filenames from: ../data/recipe/test/filenames.pickle (2000)
Load from:  ../data/recipe/captions.pickle
Traceback (most recent call last):
  File "main.py", line 146, in <module>
    algo = trainer(output_dir, dataloader, dataset.n_words, dataset.ixtoword)
  File "/media/data_dump_1/arushi_mca/GAN_try/AttnGAN/code/trainer.py", line 36, in __init__
    torch.cuda.set_device(cfg.GPU_ID)
  File "/home/arushi_mca/.conda/envs/MCAProject/lib/python2.7/site-packages/torch/cuda/__init__.py", line 292, in set_device
    torch._C._cuda_setDevice(device)
RuntimeError: cuda runtime error (101) : invalid device ordinal at /pytorch/torch/csrc/cuda/Module.cpp:59
